README.md

```diff
@@ -2,17 +2,64 @@
 
 ## Contents
 
+- [Installation](#installation)
+  - [Install Docker](#install-docker)
+  - [Install source{d} Community Edition](#install-source-d-community-edition)
 - [Usage](#usage)
   - [Defaults](#defaults)
   - [Commands](#commands)
   - [Working With Multiple Data Sets](#working-with-multiple-data-sets)
 - [Docker Compose](#docker-compose)
 
+## Installation
+
+### Install Docker
+
+_Please note that Docker Toolbox is not supported neither for Windows nor for macOS. In case that you're running Docker Toolbox, please consider updating to newer Docker Desktop for Mac or Docker Desktop for Windows._
+
+Follow the instructions based on your OS:
+
+- [Docker for Ubuntu Linux](https://docs.docker.com/install/linux/docker-ce/ubuntu/#install-docker-ce-1)
+- [Docker for Arch Linux](https://wiki.archlinux.org/index.php/Docker#Installation)
+- [Docker for macOS](https://store.docker.com/editions/community/docker-ce-desktop-mac)
+- [Docker Desktop for Windows](https://hub.docker.com/editions/community/docker-ce-desktop-windows). Make sure to read the [system requirements for Docker in windows](https://docs.docker.com/docker-for-windows/install/).
```

Comment:
`Docker in windows` -> `Docker on Windows`

Reply:
thanks! fixed and push-forced :D